.circleci/config.yml

```diff
@@ -8,13 +16,33 @@ executors:
     docker:
       - image: mbgl/android-ndk-r21:latest
     working_directory: ~/code
-
+    environment:
+      FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION: << pipeline.parameters.mapbox_navigation_native_snapshot >>
+    
 #-------------------------------
 #---------- WORKFLOWS ----------
 #-------------------------------
 workflows:
   version: 2
+  mapbox-navigation-native-validation:
```

Comment:
The goal should be to run the exact same test suite from upstream, so how about we move the `changelog-verification` , `release-snapshot`, and `release` jobs to new workflow.

This way we can just trigger the `default` workflow from upstream and we won't need to maintain the `mapbox-navigation-native-validation` that would just always have to be in sync with default.

Reply:
Decided to not move `changelog-verification` to separate workflow since it doesn't run for `main` branch.